src/otx/algorithms/detection/configs/instance_segmentation/efficientnetb2b_maskrcnn/model.py

```diff
@@ -54,8 +54,8 @@
                 type="DeltaXYWHBBoxCoder", target_means=[0.0, 0.0, 0.0, 0.0], target_stds=[0.1, 0.1, 0.2, 0.2]
             ),
             reg_class_agnostic=False,
-            loss_cls=dict(type="CrossEntropyLoss", use_sigmoid=False, loss_weight=1.0),
-            loss_bbox=dict(type="L1Loss", loss_weight=1.0),
+            loss_cls=dict(type="OrdinaryFocalLoss", gamma=1.5, loss_weight=1.0),
+            loss_bbox=dict(type="SmoothL1Loss", beta=1.0, loss_weight=1.0),
```

Comment:
This config is for supervised learning, not semisl. Is it intended patch?

Reply:
Thank you for this!